content/en/logs/log_configuration/processors.md

```diff
@@ -476,14 +476,14 @@ Use the [Datadog Log Pipeline API endpoint][1] with the following category proce
 
 Use the arithmetic processor to add a new attribute (without spaces or special characters in the new attribute name) to a log with the result of the provided formula. This remaps different time attributes with different units into a single attribute, or compute operations on attributes within the same log.
 
-A arithmetic processor formula can use parentheses and basic arithmetic operators: `-`, `+`, `*`, `/`.
+An arithmetic processor formula can use parentheses and basic arithmetic operators: `-`, `+`, `*`, `/`.
 
 By default, a calculation is skipped if an attribute is missing. Select *Replace missing attribute by 0* to automatically populate missing attribute values with 0 to ensure that the calculation is done.
 
 **Notes**:
 
 * An attribute may be listed as missing if it is not found in the log attributes, or if it cannot be converted to a number.
-* The operator `-` needs to be space split in the formula as it can also be contained in attribute names.
+* When using the operator `-`, please make sure to add spaces around it as attribute names like `start-time` may contain dashes. For example, formula has to be written as `(endTime - startTime) / 1000` OR `(end-time - start-time) / 1000`.
```

Comment:
```suggestion
* When using the operator `-`, add spaces around it to because attribute names like `start-time` may contain dashes. For example, the following formula must include spaces around the `-` operator: `(end-time - start-time) / 1000`.
```